OMCompiler/SimulationRuntime/c/simulation/solver/gbode_util.c

```
@@ -912,3 +915,200 @@ void deprecationWarningGBODE(enum SOLVER_METHOD method)
   messageCloseWarning(OMC_LOG_STDOUT);
   return;
 }
+
+SLOW_STATE_CACHE *slowStateCache_alloc(int n_stages, int n_states, double *c)
+{
+  SLOW_STATE_CACHE *cache = (SLOW_STATE_CACHE*) malloc(sizeof(SLOW_STATE_CACHE));
+  cache->n_stages = n_stages;
```

Cppcheck (2.20.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)